pkg/updater/github.go

```diff
@@ -101,16 +103,13 @@ func (g *Github) GetLatestVersion(ctx context.Context, currentVersion string, in
 	ctx = trace.StartCall(ctx, "github.GetLatestVersion", olog.F{"currentVersion": currentVersion, "prereleases": includePrereleases})
 	defer trace.EndCall(ctx)
 
-	// if we can't determine the version, fallback to empty (oldest) version
-	version := semver.MustParse("0.0.0")
-	if currentVersion != "" {
-		parsedVersion, err := semver.ParseTolerant(currentVersion)
-		if err == nil {
-			version = parsedVersion
-		}
+	version, err := semver.ParseTolerant(currentVersion)
+	if err != nil {
+		// if we can't determine the version, fallback to empty (oldest) version
+		version = semver.MustParse("0.0.0")
 	}
 
-	// Skip pre versions that don't aren't rc (made by bootstrap)
+	// Skip pre versions that aren't rc (made by bootstrap)
```

Comment:
Why though?

Reply:
Eh it's old logic that has since lost reasoning because of a lack of a comment.